transcom/mymove
# B-18322 Update Requested pickup date

## This PR was made from an already merged PR located [HERE](https://github.com/transcom/mymove/pull/11803)

## [B-18322](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a873997)

## Summary

Was kicked back from testing due to incorrect values populating. We both looked at what the correct values should be and I updated the code accordingly.
I took this opportunity to expand the test coverage a little bit as well.

## How to test

1. Make a move with PPM shipment on it.
2. As a SC user approve the move.
3. As TOO open the move and navigate to the shipments tab.
4. You should be able to observer changes there and be able to confirm that the data shown matches the data in the DB.

Planned move date should be the same date that was selected during creation of the move/shipment.
Actual move date will show only if the PPM has completed.

## Side Notes
The only commits related specifically to this issue is d49fbb0e34597491af7995791dcda8ca95c2a233 and edff35982b5c103e9a255b49f17e15d713fafd57
The others were generated due to making this branch from main and now trying to merge into integrationTesting
